### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by boqiny**

### CODE REVIEW

File: uniflow/flow/transform/model.py
Comment (goldmermaid) [inline uniflow/flow/transform/model.py:1]: Can you remove these 5 lines? We have already added it to model_op.py. ;)

Comment (boqiny) [reply uniflow/flow/transform/model.py:1]: but these are also not defined in model.py, there were errors

Comment (goldmermaid) [reply uniflow/flow/transform/model.py:1]: Actually you are right. Good catch!

**Review by goldmermaid — APPROVED** [review]
Great work @boqiny ! You can add a short description of what you did in the future, e.g. "Fixed class import class errors in ..."

**Pull request merged**